packages/admin/blocks-admin/src/blocks/YouTubeVideoBlock.tsx

```diff
@@ -23,6 +23,8 @@ const isValidYouTubeIdentifier = (value: string) => {
 };
 
 const validateIdentifier = (value?: string) => {
+    if (!value) return false;
```

Comment:
I believe this should return `undefined`. `validate` should return `undefined` if a field is valid: https://final-form.org/docs/react-final-form/types/FieldProps#validate

Reply:
https://github.com/vivid-planet/comet/pull/2121/commits/2284d077030f7f4ae28e2e4602fbb6cd982b2151